static/js/timerender.js

```diff
@@ -119,10 +119,10 @@ exports.update_timestamps = function () {
             if (element !== null) {
                 var time = elem[1];
                 var time_above;
-                var rendered_time = render_now(time);
+                var rendered_time = exports.render_now(time);
                 if (elem.length === 3) {
                     time_above = elem[2];
-                    var rendered_time_above = render_now(time_above);
+                    var rendered_time_above = exports.render_now(time_above);
```

Comment:
I merged the timerender.js piece of this, since it looked done.  Thanks @krtkmj!